content/_data/changelogs/weekly.yml

```diff
@@ -6,36 +6,40 @@
   date: 2017-02-19
   changes:
     - type: bug
-      message: Update Groovy to 2.4.8 to address memory leak issue
+      message: >
+        Update Groovy to 2.4.8 to address memory leak issue.
+        <strong>Do not use this version if you are running Pipeline builds.</strong>
       issue: 33358
+      references:
+        - issue: 42189
```

Comment:
Yeah I wanted to differentiate the _fix_ from the _regression_ somehow, but I can do it that way if you prefer.

(I think I tried that change but `./gradlew dev` seems super flaky and it just started crashing.)